app/js/components/mixins/ModalEffects.react.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+let ModalEffects = InnerComponent => class extends React.Component {
```

Comment:
@andela-iamadi I get the implementation. and like mixins.. its like an hack. we should always try to make limited use of it.